GenderPayGap.WebUI/Views/ScopeNew/ConfirmInScope.cshtml

```diff
@@ -1,9 +1,107 @@
+                Href = Url.Action("ManageOrganisation", "Organisation", new {id = encryptedOrganisationId})
+            },
+            new CrumbViewModel
+            {
+                Text = "Change Scope (Reporting Year " + reportingYearsString + ")"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
 }
+
+@{
+    string reportingYearFormatted = Model.ReportingYear.Year + "/" + (Model.ReportingYear.Year + 1).ToString().Substring(2, 2);
+    string snapshotDate = new GDSDateFormatter(Model.ReportingYear).FullStartDate;
+}
+
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        Confirm in scope
+        <form method="POST" action="@Url.Action("ConfirmInScopeAnswers", "ScopeNew", new {encryptedOrganisationId, reportingYear = Model.ReportingYear.Year })">
+            @(Html.AntiForgeryToken())
+            <h1 class="govuk-heading-xl">Confirm your organisation is required to report</h1>
+
+            <table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    @if (Model.Organisation.CompanyNumber != null)
+                    {
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header" scope="row">
+                                Company Number
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.Organisation.CompanyNumber
+                            </td>
+                        </tr>
+                    }
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            Employer Name
+                        </th>
+                        <td class="govuk-table__cell">
+                            @Model.Organisation.OrganisationName
+                        </td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">
+                            Reporting Year
+                        </th>
+                        <td class="govuk-table__cell">
+                            @reportingYearFormatted
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <p class="govuk-body">
+                Please confirm your organisation is in scope to report Gender Pay Gap data for the @reportingYearsString reporting year.
+            </p>
+
+            @(Html.GovUkButton(
+                new ButtonViewModel
+                {
+                    Text = "Confirm"
+                }
+                ))
+            
+            <p class="govuk-body">
+                <a class="govuk-link" href="@Url.Action("ManageOrganisations", "Organisation")">
+                    Cancel
+                </a>

```

Comment:
I think this *Cancel* link should take the user back to this organisation's page (rather than the organisation list page)
I think we should do this for all the Cancel buttons in these scope journeys